### ISSUE TITLE
install .NET Core Runtime the easy way in macOS via homebrew

### ISSUE THREAD

**Issue by LeeBinder** [dan0v/xdelta3-cross-gui#3]
Hi Dano. Very neat, thank you!

A little hint:
`brew cask install dotnet`
<details>
  <summary>click to expand</summary> 

```
==> Downloading https://download.visualstudio.microsoft.com/download/pr/182b16ca
######################################################################## 100.0%
==> Verifying SHA-256 checksum for Cask 'dotnet'.
==> Installing Cask dotnet
==> Running installer for dotnet; your password may be necessary.
==> Package installers may write to any location; options such as --appdir are i
Password:
installer: Package name is Microsoft .NET Core Runtime - 3.1.7 (x64)
installer: Installing at base path /
installer: The install was successful.
==> Linking Binary 'dotnet' to '/usr/local/bin/dotnet'.
🍺  dotnet was successfully installed!
```
</details>

and then later, if wanted:
`brew cask upgrade dotnet`

For those who have not yet installed homebrew yet:
`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`

:)

**Comment by dan0v** [comment]
Cheers, I'll add it to the Readme :)

**Comment by LeeBinder** [comment]
.. and how about an icon?

![Icon](https://user-images.githubusercontent.com/39203497/90559025-4ca1ab00-e19d-11ea-9f32-c78774301dd8.jpg)

[Icon Photoshop & png.zip](https://github.com/dan0v/xdelta3-cross-gui/files/5092516/Icon.Photoshop.png.zip)

:)

**Comment by dan0v** [comment]
@all-contributors please add LeeBinder for design

**Comment by LeeBinder** [comment]
Thanks @dan0v for adding me as graphics contributor and for making my quick icon sketch look much more sexy :)

**Comment by LeeBinder** [comment]
Oh, I just downloaded your new 1.0.1 macOS release and see that the icon does not show up. Icons are resource fork and must be set in macOS (does not work from Windows or Linux). I attached the improved zip with both, folder and app icon (256 x 256px 65kb to keep the size down a bit), all the same otherwise.

[xdelta3-cross-gui_1.0.1_macOS_x64-with-icon.zip](https://github.com/dan0v/xdelta3-cross-gui/files/5099200/xdelta3-cross-gui_1.0.1_macOS_x64-with-icon.zip)
